internal/test/factory/genesis.go

```diff
@@ -1,36 +1,33 @@
 package factory
 
 import (
-	"context"
-	"sort"
-	"testing"
+	"time"
 
-	"github.com/stretchr/testify/require"
-	"github.com/tendermint/tendermint/config"
-	tmtime "github.com/tendermint/tendermint/libs/time"
+	cfg "github.com/tendermint/tendermint/config"
 	"github.com/tendermint/tendermint/types"
 )
 
-func RandGenesisDoc(ctx context.Context, t *testing.T, cfg *config.Config, numValidators int, randPower bool, minPower int64) (*types.GenesisDoc, []types.PrivValidator) {
-	t.Helper()
+func GenesisDoc(
```

Comment:
this might end up rolling back some of the context/error checking changes

Reply:
This specific call doesn't really need either since it can no longer error or timeout. I took a reasonably close look to make sure I was maintaining your changes so they should still be present in this merged version.